fontforge/fvcomposite.c

```diff
@@ -2753,41 +2753,6 @@ return( ret );
 return( ret );
 }
 
-static void SCSetReasonableLBearing(SplineChar *sc,SplineChar *base,int layer) {
-    DBounds full, b;
-    SplineFont *sf;
-    int emsize;
-    double xoff;
-    RefChar *ref;
-    real transform[6];
-
-    /* Hmm. Panov doesn't think this should happen */
```

Comment:
@avpanov, take a look if you don't mind. George stubbed this function (which moves a glyph's left bearing to accommodate a protruding accent) because of you. Was it broken, or was it just being misused (called in cases when the bearing out to remain the same and the accent outside)?